src/bridge/IrcHandler.ts

```diff
@@ -217,14 +217,14 @@ export class IrcHandler {
             req.log.error("Cannot route PM to %s - no client", toUser);
             return BridgeRequestErr.ERR_DROPPED;
         }
-        req.log.info("onPrivateMessage: %s from=%s to=%s action=%s",
-            server.domain, fromUser, toUser,
-            JSON.stringify(action).substring(0, 80)
+        req.log.info("onPrivateMessage: %s from=%s to=%s",
```

Comment:
What's the reasoning to keep these on info?

Reply:
It's still really useful to keep track of when people perform actions, the info level just strips away the message data.